README.md

```diff
@@ -18,6 +18,24 @@ $ npm install react-native-simple-picker --save
 
 You will find an example in the `/exampleApp` folder.
 
+## Properties
+
+| Prop  | Default  | Type | Description | Required |
+| :------------ |:---------------:| :---------------:| :-----| :-----|
+| buttonStyle | - | `Object` | Style Close/Continue Buttons | `false` |
+| options | - | `Array` | Options that will be passed to the picker | `true`
+| initialOptionIndex | - | `Number` | Initial selected option based on it's index | `false`
+| labels | - | `Array` | Labels for the options passed to the picker | `false`
+| confirmText | Confirm | `String` | Confirm button text | `false`
+| cancelText | Cancel | `String` | Cancel button text | `false`
+| itemStyle | - | `Object` | Picker style prop. Use this to customize the picker colors, etc | `false`
+| disableOverlay | - | `bool` | When set to true it will dismiss the picker when the outside region is pressed | `false`
```

Comment:
"When set to true it will dismiss the picker when the outside region is pressed" - it is the opposite actually, if this is true it will disable the click outside of the picker... it was hard to come up with a good name.